app/src/pages/index.js

```diff
@@ -5,16 +5,38 @@ import SEO from "../components/seo"
 import BookCard from "../components/bookcard"
 
 export default ({data}) => {
+  let max=data.allBooksJson.edges.length, init=4, end=8;
+  function hide(){
+    for(var i=init; i<document.getElementsByClassName("book").length; i++){
+      document.getElementsByClassName("book")[i].hidden=true;
```

Comment:
We dont want to hide all the books. The landing page should be filled with books. Also I was thinking more in the direction of getting only enough books to fill the landing page rather than getting all the books, creating elements and then hiding them. Seems like a lot of unecessary work.